armi/reactor/reactorParameters.py

```diff
@@ -168,27 +168,46 @@ def detailedNucKeys(self, value):
 
         pb.defParam("numMoves", units="", description="numMoves", default=0)
 
-    with pDefs.createBuilder(default=0.0, location="N/A") as pb:
+    with pDefs.createBuilder(
+        location="N/A", default="None", categories=["control rods"]
+    ) as pb:
 
         pb.defParam(
-            "breedingRatio2",
-            units="N/A",
-            description="Ratio of fissile Burned and discharged to fissile discharged",
-            saveToDB=False,
+            "crMostValuablePrimaryRodLocation",
+            units=None,
+            saveToDB=True,
+            description=(
+                "Core assembly location for the most valuable primary control rod."
+            ),
+        )
+        pb.defParam(
+            "crMostValuableSecondaryRodLocation",
+            units=None,
+            saveToDB=True,
+            description=(
+                "Core assembly location for the most valuable secondary control rod."
+            ),
         )
-
         pb.defParam(
             "crWorthRequiredPrimary",
-            units="$",
+            units="pcm",
             description="The total worth in $ required for primary control rods to shutdown reactor accounting for uncertainties and margins",
```

Comment:
description says $ which is now inconsistent with units. recommend just removing the `in __` clause to avoid possiblity.